src/pages/overview/pdf-accessibility-auto-tag-api/howtos/accessibility-auto-tag-api.md

```diff
@@ -79,27 +79,33 @@ public class AutotagPDF {
     public static void main(String[] args) {
```

Comment:
> Is this an as-is copy of the samples 

No samples intent to segregate since user can run multile samples at once but this is a quick start.

we are mentioning samples so user can go and use it as reference as well.